src/libsystemd-network/radv-internal.h

```diff
@@ -171,6 +177,32 @@ struct sd_radv_route_prefix {
         usec_t valid_until;
 };
 
+/* rfc8781: section 4 - Scaled Lifetime: 13-bit unsigned integer. PLC (Prefix Length Code): 3-bit unsigned integer */
+#define radv_pref64_prefix_opt__contents { \
+        uint8_t type;                      \
+        uint8_t length;                    \
+        uint16_t lifetime_prefixlen;       \
```

Comment:
I'd like to rename `lifetime_prefixlen` -> `lifetime_and_plc` or `lifetime_and_prefixlen_code` so, as it does not directly store prefixlen.